Tests/TuistKitTests/ProjectEditor/ProjectEditorMapperTests.swift

```diff
@@ -34,40 +34,55 @@ final class ProjectEditorMapperTests: TuistUnitTestCase {
                                            projectDescriptionPath: projectDescriptionPath)
 
         // Then
-        let manifestsTarget = project.targets.first
-        let helpersTarget = project.targets.last
-        let expectedManifestsTarget = Target(name: "Manifests",
-                                             platform: .macOS,
-                                             product: .staticFramework,
-                                             productName: "Manifests",
-                                             bundleId: "io.tuist.${PRODUCT_NAME:rfc1034identifier}",
-                                             settings: expectedSettings(sourceRootPath: sourceRootPath),
-                                             sources: manifestPaths.map { (path: $0, compilerFlags: nil) },
-                                             filesGroup: .group(name: "Manifests"),
-                                             dependencies: [.target(name: "ProjectDescriptionHelpers")])
-        let expectedHelpersTarget = Target(name: "ProjectDescriptionHelpers",
-                                           platform: .macOS,
-                                           product: .staticFramework,
-                                           productName: "ProjectDescriptionHelpers",
-                                           bundleId: "io.tuist.${PRODUCT_NAME:rfc1034identifier}",
-                                           settings: expectedSettings(sourceRootPath: sourceRootPath),
-                                           sources: helperPaths.map { (path: $0, compilerFlags: nil) },
-                                           filesGroup: .group(name: "Manifests"),
-                                           dependencies: [])
-        let expectedProject = Project(path: sourceRootPath,
-                                      name: "Manifests",
-                                      settings: Settings(base: [:],
-                                                         configurations: Settings.default.configurations,
-                                                         defaultSettings: .recommended),
-                                      filesGroup: .group(name: "Manifests"),
-                                      targets: [expectedManifestsTarget, expectedHelpersTarget])
-        XCTAssertEqual(project, expectedProject)
```

Comment:
This test was very granular and tested ALL things. It was hard to tell why the check fails basically meaning we have to copy most of the code from `ProjectEditorMapper` to make it pass.